src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/code/StubQueue.java

```diff
@@ -52,8 +52,8 @@ public class StubQueue extends VMObject {
   private static CIntegerField queueEndField;
   private static CIntegerField numberOfStubsField;
 
-  // The type of the contained stubs (i.e., InterpreterCodelet,
-  // ICStub). Must be a subclass of type Stub.
+  // The type of the contained stubs (i.e., InterpreterCodelet).
+  // Must be a subclass of type Stub.
```

Comment:
This is interesting. The StubQueue seems to work against an interface and that interface used to have two implementing classes. Now that one of the is gone, we might want to reconsider if we need this interface. There seems to be a lot of code that could be remove in this area. See `DEF_STUB_INTERFACE` and how it is being used.